lips/lip-1.mdx

```diff
@@ -579,34 +688,31 @@ The `receiver_metadata_signature` will then be encoded in hex and placed into th
 Once both sender and receiver have a status of 'ready_for_settlement', the transaction may then be submitted on-chain by the sender VASP. This submission will utilize the `recipient_signature` which was provided by the receiver VASP (generated via [Recipient Signature](#recipient-signature).  The sender VASP will now generate a transaction via the following and then submit the transaction on-chain:
 
 ```
-use libra_types::transaction::{
-    RawTransaction,
-    metadata::{Metadata, TravelRuleMetadata, TravelRuleMetadataV0}
-};
+from diem import txnmetadata, stdlib, utils, diem_types
 
-metadata = Metadata::TravelRuleMetadata(
-  TravelRuleMetadata::TravelRuleMetadataVersion0(
-    TravelRuleMetadataV0 {
-      off_chain_reference_id: "123abc", /* The off-chain reference ID specified by `reference_id` in the Off-chain APIs */
-}));
+reference_id = "bb991d8e3e6011eb8eaeacde48001122"
+sender_account_address = utils.account_address("793251de1a61e9b4d1a17d13aa015e45")
+amount = 1_000_000_000
 
+lcs_metadata, _ = txnmetadata.travel_rule(reference_id, sender_account_address, amount)
 
-lcs_metadata = lcs.serialize(metadata, Metadata);
-
-program = encode_peer_to_peer_with_metadata_script(
-    "LBR" /*currency*/,
-    0x1234 /*recipient*/,
-    100 /*amount*/,
-    lcs_metadata,
-    receiver_metadata_signature /* from off-chain API exchange */);
+script = stdlib.encode_peer_to_peer_with_metadata_script(
+    currency=utils.currency_code("XUS"),
+    payee=receiver_account_address,
+    amount=amount,
+    metadata=lcs_metadata,
+    metadata_signature=receiver_metadata_signature,
+)
 
-RawTransaction {
-    sender_account: 0x7777,
-    program: program,
+diem_types.RawTransaction(
+    sender=sender_account_address,
+    payload=diem_types.TransactionPayload__Script(script),
+    ...
 }
 ```
 
-For additional details, see https://lip.libra.org/lip-4/#c-to-c-transaction-flow
+For additional details, see https://dip.diem.com/dip-4/#c-to-c-transaction-flow or https://github.com/diem/client-sdk-python/blob/master/examples/p2p_transfer.py#L127
+
 
 # Reference Implementation
 A reference implementation of the Off-Chain Protocol is located at https://github.com/libra/off-chain-reference.
```

Comment:
Is this still the URL or has it been updated for diem?

Reply:
diem => https://github.com/diem/off-chain-reference